tests/resources/defects.json

```diff
@@ -2214,7 +2217,10 @@
         "location": {
           "vertical": null,
           "horizontal": null,
-          "lateral": null,
+          "lateral": [
+            "nearside",
```

Comment:
ticket suggests nearside/offside/center values?

Reply:
You are right, the json file has changed but I think Bogdan did not update the ticket description.

Reply:
Ok, you were right @KHodderDVSA . I have discussed it with Carolina and the json file was missing the "centre" value. I updated it on the ticket. @m-ujaffer can you please update defects.json and run all the tests?

Reply:
Updated the defects.json file. ran the unit and integration tests successfully.